test/test_dependencies.jl

```diff
@@ -0,0 +1,47 @@
+using Mimi
+
+#list of URLs of branches of packages to test
+dependencies = ["https://github.com/davidanthoff/fund.jl/archive/master.zip"]
+
+#list of failed tests to build as you go
+errors = []
+
+#make a temporary directory to run the tests in
+run(`mkdir tmp_testing`)
```

Comment:
We should probably not shell out for file system operations because this will probably not work cross-platform. Better to use the julia filesystem functions http://docs.julialang.org/en/release-0.4/stdlib/file/.